app.py

```diff
@@ -110,7 +110,7 @@ def get_dcv_session_():
         return get_dcv_session()
 
     @app.route("/manager/get_identity")
-    @authenticated({"guest"})
+    @authenticated(ADMINS_GROUP)
```

Comment:
suggestion(non-blocking): since almost all routes now are accessible only by admin users, it would be nice to introduce authentication by default on all endpoints, and allow access from unregistered users only on authentication methods

Reply:
Agree. Let's do it on a separate PR, so that we can focus on properly test such change